src/map/utils/petutils.cpp

```diff
@@ -1596,7 +1596,8 @@ namespace petutils
             PPet->SetMLevel(PMaster->GetMLevel());
             PPet->health.maxhp = (uint32)floor((250 * PPet->GetMLevel()) / 15);
             PPet->health.hp    = PPet->health.maxhp;
-
+            // This sets the correct visual size for the luopan
+            PPet->m_flags      = 0x0000008B;
```

Comment:
so the reason this wasn't loaded from sql is it was never added to the query for building the entity:
https://github.com/LandSandBoat/server/blob/base/src/map/utils/petutils.cpp#L140

Reply:
it would take it from mob pools normally by being added to that and using another handle entry below the linked section to populare PPet->m_flags

Reply:
adjust the comment here to mention it

Reply:
Made a note of the entity flags not being used with a TODO